docs/schema.json

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
Would `validation/schema.json` being a symlink break existing users depending on the  https://raw.githubusercontent.com/ossf/osv-schema/main/validation/schema.json link ?

What if we made the docs/schema.json a symlink instead to this instead?

Reply:
I just checked and it doesn't work 😢 https://raw.githubusercontent.com/JamieMagee/osv-schema/github-pages-schema/validation/schema.json

Unfortunately doing it the other way round doesn't work either. What do you think about duplicating the file? Or just having it in `docs`?

Reply:
darn :/

Re "other way around doesn't work": do you mean having `validation/schema.json` be the source of truth, and having `docs/schema.json` as a symlink doesn't work either when it comes to  the GitHub site generation? 

Interestingly, it seems to work here: https://github.com/s4y/gh-pages-symlink-test